### ISSUE TITLE
Apigen 5

### ISSUE THREAD

**Issue by RodriAndreotti** [phingofficial/phing#978]
The integration with the problem with version 5.0 of ApiGen, apparently version 5 no longer has the source option, having changed the basic syntax to:
`apigen generate <source> --destination = <destination>`

Could I try to develop a fix for this and perform a pull request?

In case I would try to maintain backward compatibility

**Comment by kenguest** [comment]
Please do, and there's no need to ask :-) Thanks for wanting to help!

**Comment by RodriAndreotti** [comment]
Anytime!

I just found out that there are some other options too, which has problem like the --title.

I am trying to get the new syntaxes in the documentation (which is very scarce) and in the source code of ApiGen, and so already do all the correction;)

**Comment by RodriAndreotti** [comment]
Well, guys!

I made some changes here, but I ended up noting that ApiGen version 5 is full of bugs yet, and seems a bit abandoned too, many options that were passed via command-line parameters are no longer there (in fact there are now two or three only ).
The rest is done through a configuration file, but even creating this file, apigen was unable to generate the documentation for my project, so I do not know if it's worth the pull request at the moment, if you find the post interesting.
